styles.css

```diff
@@ -392,3 +393,93 @@ span {
   border: none;
   border-bottom: 1px solid  #dfe1e6;
 }
+
+.contact-me {
+  grid-area: 5 / 1 / 6 / 2;
+  background-image: url(images/contactme-shapes.png), url(images/Contactme-bg.png);
+  background-color: white;
+  background-size: contain, cover;
+  background-repeat: no-repeat, no-repeat;
+  background-position: right, left top;
+  padding: 10px 24px 0 24px;
+  flex-wrap: wrap;
+}
+
+h4 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #fff;
+  margin-top: 0;
+  margin-bottom: 12px;
+  padding-top: 103px;
+}
+
+.p-contact-me {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-align: justify;
```

Comment:
- [ ] **[OPTIONAL]** Kindly use `text-align: center` to stick to the design as much as possible. 📑 

![text-center](https://user-images.githubusercontent.com/58642949/186297570-e52e726a-a31a-4cdc-b7b7-9d79ada325f4.png)